Soft Allocation Journey.md

```diff
@@ -0,0 +1,193 @@
+# Introduction to Soft Allocation Brokering
+
+Hotwax Commerce's Soft Allocation Brokering feature performs real-time eligibility checks for same-day delivery of items in a customer's shopping cart. By evaluating the list of items in the cart along with the customer's zip code, Soft Allocation Brokering ensures compliance with predefined criteria for same-day shipping. 
+
+This process guarantees a seamless and efficient validation of same-day shipping eligibility, aligning with the requirements of various shipping carriers.
+
+Internally, the Soft Allocation process adheres to a structured set of steps: 
+
+
+### Step 1: Obtain the customer's latitude and longitude
+
+To check eligible same-day shipping facilities near the customer, the latitude and longitude coordinates of the customer are required. There are two ways to obtain these coordinates:
+
+a. If the customer inputs their postal code,
+  - Use the `postcodeLookup` API to convert the postal code into latitude and longitude coordinates.
+
+b. If the customer allows access to their system location,
+  - Obtain the latitude and longitude coordinates directly from their device.
+    
+
+### Sample of `postCodeLookup` :
+
+#### Method : `/POST`
+
+## Request
+
+```
+{
+  "json": {
+    "params": {
+      "q": "*:*",
+      "fq": "{!geofilt sfield=location}",
+      "pt": "18.180555,-66.749961",
+      "d": "10"
+    }
+  }
+}
+```
+
+## Response
+
+```
+{
+  "responseHeader": {
+    "status": 0,
+    "QTime": 2,
+    "params": { "json": "{\"query\":\"postcode:10007\"}"
+  }
+},
+  "response": {
+    "numFound": 1,
+    "start": 0,
+    "numFoundExact": true,
+    "docs": [
+  {
+        "postcode": "10007",
+        "latitude": "40.713858",
+        "longitude": "-74.007777",
+        "country": "USA",
+        "country-code-id": "10007-USA",
+        "location": "40.713858,-74.007777",
+        "\_version\_": 1752733529047826432
+      }
+    ]
+  }
+}
+```
+
+## Step 2: Verify store inventory for same-day delivery of cart items.
+
+Utilize the `checkCartInventory` API to determine same-day delivery eligibility for cart items across stores. This API evaluates eligibility based on key factors, including facility latitude and longitude and a predefined distance (typically within 50 miles) for same-day delivery by the carrier. The results indicate the closest store eligible for same-day delivery.
+
+### Sample of `checkCartInventory`:
+
+### Method : `/POST`
+
+## Request
+
+```
+{
+  point: "\\<latlon\\>",
+  distance: "sameDayStoreSearchProximity",
+  shopifyShopId: "shopId",
+  items: "cartItems",
```

Comment:
If this is where the skus are listed we should give an example with multiple items so that the reader knows how to delimit them.